### DISCUSSION TITLE
styling disabled buttons

### DISCUSSION THREAD

**Discussion by rishavs** [saadeghi/daisyui#2675] [General]
Is there a way to style disabled buttons?
I have a disabled button
`<button class="btn btn-disabled">DISABLED</button>`
I'd like to be able to customize the text in there, for eg. `<button class="btn btn-disabled text-warning">DISABLED</button>`,
but nothing seems to work on disabled buttons.

**Comment by saadeghi** [comment]
`btn-disabled` class overrides all the button styles as it should because if we add/remove this class name to any button it is expected to apply/remove the disabled styles from the button so it has a higher CSS specificity.

To override this style, you can use something like `[.btn&]:text-warning`:  
https://play.tailwindcss.com/iycutydKii

Let me know if you have a question.

**Reply by rishavs** [reply]
Thank you @saadeghi
